www/doc/1.7/report.html

```diff
@@ -20,143 +22,186 @@ <h1>Herbie reports</h1>
 
   <h2 id="summary">Summary numbers</h2>
 
-  <p>First, a brief summary of the results. For most uses, the
-    “Average Error” number, which describes how accurate the input and
-    output expressions are, is the most important statistic in this
-    section. The other numbers list time Herbie took to improve the
-    program and the <a href="input.html#precisions">precision</a>
-    assumed for floating-point operations.
-  </p>
+  <p>First, a brief summary of the results. Herbie can produces multiple 
+    alternatives to the given program.</p>
 
   <figure>
     <img width="100%" src="report-large.png" />
-    <figcaption>Summary numbers from a Herbie report. The "binary64"
-    precision refers to double-precision IEEE-754
-    arithmetic.</figcaption>
+    <figcaption>Summary numbers from a Herbie report.</figcaption>
   </figure>
 
-  <h2 id="programs">Input and output programs</h2>
+  <ul>
+    <li><b>Percentage Accurate</b>: The change in accuracy from the initial 
+      program to what Herbie thinks might be the most accurate alternative.
+    </li>
+    <li><b>Time</b>: The time it took Herbie to improve the given program.</li>
+    <li><b>Alternatives</b>: Number of alternative solutions found.</li>
+    <li><b>Speedup</b>: Relative speed of fastest alternative that improves accuracy.</li>
+  </ul>
+
+  <h2 id="spec">Specification</h2>
+
+  <p>Second, a summary of the specification that you gave to Herbie.
+    Typically, the specification is also the initial program, but in some cases, like if the <a href="http://fpbench.org/spec/fpcore-2.0.html#syntax" target="_blank">FPCore</a> property <kbd>:spec</kbd> is used, they can differ.
+    <a href="input.html#preconditions">Preconditions</a> may also be added to give Herbie more information about the expression you provided. Please see the
+    <a href="input.html#sec1">input page</a> for more information.</p>
 
-  <p>Second, the input and output programs themselves, in standard
-  mathematical syntax. In the top-right corner, the drop-down can be
-  used to switch to C syntax or some other format.</p>
+  <p>In the top right you can change the notation of the expression from Math 
+    to other syntax like FPCore or Python.</p>
 
+  <p>The colored outcome bar summarizes the sampled floating-point inputs that
+    produce valid, unknown, or invalid outputs. Green outcomes are valid, broken down into finite, infinite, and zero outcomes. Unknown outputs are red. Blue outcomes are invalid (fail precondition or domain errors) and are ignored by Herbie.</p>
+
+  <!-- Colors copied from src/web/resources/report.css -->
   <figure>
-    <img width="100%" src="report-prog.png" />
-    <figcaption>Input and output program from a Herbie report.</figcaption>
+    <img width="100%" src="specification.png" />
+    <figcaption>Summary of the specification and outcomes bar
+      <ul>
+        <li><b style="color : #4e9a06;">valid</b>: Successfully sampled</li>
```

Comment:
@pavpanchekha  Let me know what you think of this. I know it's missing `unknown` but also wanted to circle back on renaming `valid` to `finite` and adding `zero` outcome?

https://usocgrad.slack.com/archives/C0G44MH4P/p1687196931068189?thread_ts=1687192104.957769&cid=C0G44MH4P

Reply:
I think this summary should go in the main text, and a shorter summary in the caption. I don't think `unknown` can actually happen in practice? (I think it's a temporary state after analysis but before sampling.)

I think splitting `valid` into `finite` and `zero` is nice, but you should make a separate PR where you both implement that and update the docs to describe it.